sys-auth/google-oslogin/google-oslogin-20180611.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Components to support Google Cloud OS Login. This contains bits that belong in USR"
+HOMEPAGE="https://github.com/GoogleCloudPlatform/compute-image-packages/tree/master/google_compute_engine_oslogin"
+SRC_URI="https://github.com/GoogleCloudPlatform/compute-image-packages/archive/${PV}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="amd64"
+IUSE=""
+
+inherit pam toolchain-funcs
+
+DEPEND="
+	net-misc/curl[ssl]
+	dev-libs/json-c
+	virtual/pam
+"
+
+RDEPEND="${DEPEND}"
+
+S=${WORKDIR}/compute-image-packages-${PV}/google_compute_engine_oslogin
+
+src_prepare() {
+	eapply -p2 "$FILESDIR/0001-pam_module-use-var-lib-instead-of-var.patch"
```

Comment:
Is there a reason to do this instead of setting `$PATCHES` and adapting the patch to work with `-p1`?

Reply:
Mostly because this makes generating the patch easy.